### PR TITLE
Implement type 2033, PixelMonitoring, fixes #243

### LINKED ISSUE #243: Support for type 2033, pixel monitoring
Quick prototyping:

```
from eventio import EventIOFile
from eventio.base import EventIOObject
for e in EventIOFile('./Data/sim_telarray/cta-prod5-lapalma/0.0deg/Data/gamma_20deg_180deg_run6___cta-prod5-lapalma-2158m-LaPalma-dark.simtel.zst'):
    if type(e) is EventIOObject:
        break
        
from io import BytesIO

stream = BytesIO(e.read())

from eventio import tools as io
flags = io.read_int(stream)
num_pixels = io.read_int(stream)
num_gains = io.read_short(stream)

print(flags, num_pixels, num_gains)

import numpy as np

nsb_rate = io.read_array(stream, np.float32, num_pixels)
nsb_rate = io.read_array(stream, np.float32, num_pixels)
qe_rel = io.read_array(stream, np.float32, num_pixels)
gain_rel = io.read_array(stream, np.float32, num_pixels)
hv_rel = io.read_array(stream, np.float32, num_pixels)
current = io.read_array(stream, np.float32, num_pixels)
fadc_amp_highgain = io.read_array(stream, np.float32, num_pixels)
fadc_amp_lowgain = io.read_array(stream, np.float32, num_pixels)
disabled = io.read_array(stream, np.uint8, num_pixels)
```

### PR DESCRIPTION
**Pull request by maxnoe**

### CODE REVIEW

File: tests/simtel/test_simtel_objects.py [hunk 1]
```diff
@@ -724,3 +724,28 @@ def test_2032():
...
+def test_2033():
+    from eventio.simtel.objects import PixelMonitoring
+
+    with EventIOFile('tests/resources/type2033.simtel.zst') as f:
+        objects = yield_n_and_assert(f, PixelMonitoring, n=19)
+        tel_id = 0
+        for tel_id, o in enumerate(objects, start=1):
+            assert o.header.id == tel_id
+            data = o.parse()
+            assert data.keys() == {
```
Comment (orelgueta) [inline tests/simtel/test_simtel_objects.py:738]: The `fadc_amp_lg` key would not be there if the test file does not have 2 gains. Maybe it's better to make the test more general?

File: tests/simtel/test_simtelfile.py [hunk 2]
```diff
@@ -237,3 +239,13 @@ def test_history_meta():
         assert isinstance(f.telescope_meta, dict)
         assert len(f.global_meta) == 11
         assert len(f.telescope_meta) == 19
+
+
+def test_type_2033():
+    with SimTelFile(pixel_monitoring_path) as f:
+        e = next(iter(f))
+        assert e['pixel_monitorings'].keys() == e['telescope_events'].keys()
+        assert e['pixel_monitorings'][1].keys() == {
```
Comment (orelgueta) [inline tests/simtel/test_simtelfile.py:248]: Same as above about the number of gains, though here I am not sure cause I cannot tell without digging a bit if the keys are filled in the parsing or somewhere else.

**Review by orelgueta — COMMENTED** [review]
The tests are dependent on the file being for a camera with 2 gains. Maybe it's OK, but it's perhaps better to make the test work also for a single gain camera.

File: tests/simtel/test_simtel_objects.py (hunk 1 above)
Comment (maxnoe) [reply tests/simtel/test_simtel_objects.py:738]: To limit the size of the test files, I produced a la palma run, but you are right, we should have both cases.

File: tests/simtel/test_simtelfile.py (hunk 2 above)
Comment (maxnoe) [reply tests/simtel/test_simtelfile.py:248]: Cameras with just a single gain shouldn't have the `fadc_amp_lg` key, but the test file I produced has only lst and mst nectarcam. I will produce a test file with paranal.

**Comment by maxnoe** [comment]
@orelgueta I exchanged the test file for a file containing telescopes with both one and two gains

**Review by orelgueta — APPROVED** [review]
Looks good now!

**Pull request merged**